voice/player.js

```diff
@@ -153,8 +153,12 @@ module.exports = function Player (text, url, transient, sequential, tag) {
     if (focus.player == null) {
       return
     }
+    focus.player.once('seekcomplete', () => {
```

Comment:
Is there any event like `seekerror` need to be handled?

Reply:
Doesn't seem to have one like that.